basyx.aasregistry/basyx.aasregistry-client-native/src/main/java/org/eclipse/digitaltwin/basyx/aasregistry/main/client/mapper/AttributeMapper.java

```diff
@@ -146,4 +149,80 @@ public AssetKind mapAssetKind(org.eclipse.digitaltwin.aas4j.v3.model.AssetKind a
 		}
 	}
 
+	/**
+	 * Maps {@link AssetInformation#getSpecificAssetIds()} from AAS4J to AasRegistry client
+	 *
+	 * @param specificAssetIds
+	 * @return the mapped specificAssetIds
+	 */
+	public List<SpecificAssetId> mapSpecificAssetIds(List<org.eclipse.digitaltwin.aas4j.v3.model.SpecificAssetId> specificAssetIds) {
+		if (specificAssetIds == null || specificAssetIds.isEmpty()) {
```

Comment:
You can remove the curly braces if there is just a single statement in the if block.